easymotion.kak

```diff
@@ -112,7 +112,7 @@ pydef 'easy-motion-on-selections -params 0..3' '%opt{em_jumpchars}^%val{timestam
                 break
             out += [q(c), q(dfs(chars_c))]
         return ' '.join(out + [q(';easy-motion-rmhl;' + callback_cancel)])
-    return "\n".join((
+    return "\\n".join((
```

Comment:
Hmm, adding this actually breaks the plugin when I test on my linux machine... Seems there might be a difference in how escaping is handled between different OSes, unfortunately. Will look into this and see if I can figure this out later.

Reply:
Its a difference between `echo` on different platforms. I've switched to using printf.